observations/obs_converters/GSI2DART/mpisetup.f90

```diff
@@ -78,7 +59,6 @@ subroutine mpi_initialize()
    print *,'ERROR: illegal r_kind (must be single or double)'
    call mpi_cleanup()
 endif
-my_communicator = mpi_comm_world
 
```

Comment:
i would leave the my_communicator line in.  it makes it easier in the future to create a private communicator if that becomes needed.